command/commands.go

```diff
@@ -142,6 +143,7 @@ var (
 		"manta":                  physManta.NewMantaBackend,
 		"mssql":                  physMSSQL.NewMSSQLBackend,
 		"mysql":                  physMySQL.NewMySQLBackend,
+		"oci_objectstorage":      physOCI.NewBackend,
```

Comment:
I think simply "oci" would be more consistent with e.g. "gcs", "azure", unless you expect another oci type of storage to be added?